.github/workflows/update-core-example.yml

```diff
@@ -0,0 +1,19 @@
+name: Update ChainBridge core-example
+
+on:
+  push:
+    branches:
+      - 'test-cd'
+
+jobs:
+  update-core-example:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger chainbridge-core-example update event
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          # Dispatch event from current repository
+          repository: github.repository
```

Comment:
isn't it should be repo name?

Reply:
`github.repository` means the event dispatched from the current repository, that is `chainbridge-core`